raft/fsm.hh

```diff
@@ -41,14 +41,6 @@ struct fsm_output {
     bool state_changed = false;
     // Set to true if a leadership transfer was aborted since the last output
     bool abort_leadership_transfer;
-
```

Comment:
Well, someone did try to keep it up to date :) https://github.com/scylladb/scylladb/issues/10912